src/link/rgblink.5

```diff
@@ -0,0 +1,63 @@
+.Dd March 3, 2017
+.Dt RGBLINK 5
+.Os RGBDS Manual
+.Sh NAME
+.Nm rgblink
+- linkerscript file format
+.Sh DESCRIPTION
+The linkerscript is an external file that allows the user to specify the
+order of sections without the need for doing so before assembling each object
+file.
+.Pp
+The placement of sections specified in the linkerscript is done before the
+sections whose placement is defined in the source code.
+.Pp
+A linkerscript consists on a series of banks followed by a list of sections
+and, optionally, commands. They can be lowercase or uppercase, it is ignored.
+Any line can contain a comment starting with ';' that ends at the end of the
+line:
+.Pp
+  ROMX $F ; This is a comment
+   "Functions to read array"
+   ALIGN 8
+   "Array aligned to 256 bytes"
+
+  WRAMX 2
+    "Some variables"
+.Pp
+Numbers can be in decimal or hexadecimal format (the prefix is '$'). It is an
+error if any bank or command is found before setting a bank.
+.Pp
+The possible bank types are: ROM0, ROMX, VRAM, WRAM0, WRAMX, OAM and HRAM.
+Types ROMX, VRAM, WRAMX and SRAM are banked, which means that it is needed to
+specify a bank after the type.
+.Pp
+When a new bank statement is found, sections found after it will be placed
+right from the beginning of that bank. If the linkerscript switches to a
+different bank and then it comes back to the previous one it will continue
+from the last address that was used.
+.Pp
+The only two commands are ORG and ALIGN:
+.Pp
```

Comment:
Begin List/End List:

    The only two commands are ORG and ALIGN:
    .Bl -bullet
    .It
    ORG sets the address…
    .It
    ALIGN will increase…
    .El